### DISCUSSION TITLE
Can OperationContext.Current.Channel be changed from internal to public?

### DISCUSSION THREAD

**Discussion by actionthomas** [CoreWCF/CoreWCF#1224] [Q&A]
Hi

I'm migrating yet another service in .net 7 and saw that `OperationContext.Current.Channel` is now internal.
In the [source code](https://github.com/CoreWCF/CoreWCF/blob/ac5453e5520db73132e2739d536078e64c90ded1/src/CoreWCF.Primitives/src/CoreWCF/OperationContext.cs#L72), there's this comment: 
`// TODO: Probably want to revert this to public`
Does anyone know why it's been changed at the time, and if there's a risk in exposing the property again?

Thanks!

**Comment by icnocop** [comment]
See https://github.com/CoreWCF/CoreWCF/discussions/1144#discussioncomment-11488430
